stdlib/messaging/rabbitmq/src/main/java/org/ballerinalang/messaging/rabbitmq/observability/RabbitMQObservabilityConstants.java

```diff
@@ -0,0 +1,91 @@
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.messaging.rabbitmq.observability;
+
+/**
+ * Constants for RabbitMQ Observability.
+ *
+ * @since 1.2.0
+ */
+public class RabbitMQObservabilityConstants {
+
+    static final String CONNECTOR_NAME = "rabbitmq";
+
+    static final String[] METRIC_CONNECTIONS = {"connections", "Number of currently active connections"};
+    static final String[] METRIC_CHANNELS = {"channels", "Number of currently active channels"};
+    static final String[] METRIC_QUEUES = {"queues", "Number of currently active queues"};
+    static final String[] METRIC_EXCHANGES = {"exchanges", "Number of currently active exchanges"};
+    static final String[] METRIC_CONSUMERS = {"consumers", "Number of currently active consumers"};
+    static final String[] METRIC_PUBLISHED = {"published", "Number of messages published"};
+    static final String[] METRIC_PUBLISHED_SIZE = {"published_size", "Total size in bytes of messages published"};
+    static final String[] METRIC_ERRORS = {"errors", "Number of errors"};
+    static final String[] METRIC_SUBSCRIPTION = {"subscriptions", "Number of subscriptions"};
+    static final String[] METRIC_CONSUMED = {"consumed", "Number of messages consumed"};
+    static final String[] METRIC_CONSUMED_SIZE = {"consumed_size", "Total size in bytes of messages consumed"};
+    static final String[] METRIC_ACK = {"acknowledgements", "Total number of acknowledgements received"};
+
+    static final String TAG_URL = "url";
+    static final String TAG_ERROR_TYPE = "error_type";
+    public static final String TAG_QUEUE = "queue";
+    static final String TAG_CHANNEL = "channel";
+    static final String TAG_EXCHANGE = "exchange";
+    static final String TAG_ROUTING_KEY = "routing_key";
+    static final String TAG_ACK_TYPE = "ack_type";
+    static final String TAG_CONSUME_TYPE = "consume_type";
+
+    public static final String ERROR_TYPE_CONNECTION = "connection";
+    public static final String ERROR_TYPE_CONNECTION_CLOSE = "connection_close";
+    public static final String ERROR_TYPE_CONNECTION_ABORT = "connection_abort";
+    public static final String ERROR_TYPE_PUBLISH = "publish";
+    public static final String ERROR_TYPE_CONSUME = "consume";
+    public static final String ERROR_TYPE_CHANNEL_CREATE = "channel_create";
+    public static final String ERROR_TYPE_CHANNEL_CLOSE = "create_channel";
```

Comment:
I believe you meant to say `channel_close` here :) 
```suggestion
    public static final String ERROR_TYPE_CHANNEL_CLOSE = "channel_close";
```